src/extension.ts

```diff
@@ -66,6 +58,20 @@ export async function activate(context: ExtensionContext) {
 		return;
 	}
 
+    // Load save data
+	const resSave = loadData();
```

Comment:
This HAS to be under the resource DB to make sure it doesn't try to load user preferences against assets which don't even exist yet